src/ko/wolfdotcom/src/eu/kanade/tachiyomi/extension/ko/wolfdotcom/Wolf.kt

```diff
@@ -0,0 +1,398 @@
+                        SManga.create().apply {
+                            url = mangaUrl.queryParameter("toon")!!
+                            title = el.selectFirst(".searchDetailTitle")!!.text()
+                            thumbnail_url = el.selectFirst(".searchPng")
+                                ?.attr("style")
+                                ?.let {
+                                    styleImage.find(it)?.groupValues?.get(1)
+                                }
+                        }
+                    }
+
+                MangasPage(entries, false)
+            }
+    }
+
+    override fun getMangaUrl(manga: SManga): String {
+        return "$baseUrl/$entryPath?toon=${manga.url}"
+    }
+
+    override fun mangaDetailsRequest(manga: SManga): Request {
+        return GET(getMangaUrl(manga), headers)
+    }
+
+    override fun mangaDetailsParse(response: Response): SManga {
+        val document = response.asJsoup()
+
+        return SManga.create().apply {
+            title = document.selectFirst(".text-box h1")!!.text()
+            thumbnail_url = document.selectFirst(".img-box img")?.absUrl("src")
+            description = document.selectFirst(".text-box .txt")?.text()
+            genre = document.selectFirst(".text-box .sub:has(> strong:contains(장르))")?.ownText()?.replace("/", ", ")
+            author = document.selectFirst(".text-box .sub:has(> strong:contains(작가))")?.ownText()?.replace("/", ", ")
+        }
+    }
+
+    override fun chapterListRequest(manga: SManga): Request {
+        return mangaDetailsRequest(manga)
+    }
+
+    @Serializable
+    class ChapterUrl(
+        val toon: String,
+        val num: String,
+    )
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val document = response.asJsoup()
+
+        return document.select(".webtoon-bbs-list a.view_open").map { el ->
+            val chapUrl = el.absUrl("href").toHttpUrl()
+            SChapter.create().apply {
+                url = json.encodeToString(
+                    ChapterUrl(
+                        chapUrl.queryParameter("toon")!!,
+                        chapUrl.queryParameter("num")!!,
+                    ),
+                )
+                name = el.selectFirst(".subject")!!.ownText()
+                chapter_number = el.selectFirst(".num")!!.ownText().toFloat()
+                date_upload = el.selectFirst(".date")?.text().parseDate()
+            }
+        }
+    }
+
+    private val dateFormat = SimpleDateFormat("yyyy-MM-dd", Locale.ENGLISH)
+
+    private fun String?.parseDate(): Long {
+        this ?: return 0L
+
+        return try {
+            dateFormat.parse(this)!!.time
+        } catch (_: ParseException) {
+            0L
+        }
+    }
+
+    override fun getChapterUrl(chapter: SChapter): String {
+        val chapUrl = json.decodeFromString<ChapterUrl>(chapter.url)
+        return "$baseUrl/$readerPath?toon=${chapUrl.toon}&num=${chapUrl.num}"
```

Comment:
Consider `HttpUrlBuilder` - even if you know the values can't contain invalid characters, it is better to separate concerns a bit.